core/internal/features/ocr2/features_ocr2_test.go

```diff
@@ -313,8 +313,37 @@ fromBlock = %d
-codec = '{"chainCodecConfig" :{"MedianReport":{"TypeAbi": "[{\"Name\": \"Timestamp\",\"Type\": \"uint32\"},{\"Name\": \"Observers\",\"Type\": \"bytes32\"},{\"Name\": \"Observations\",\"Type\": \"int192[]\"},{\"Name\": \"JuelsPerFeeCoin\",\"Type\": \"int192\"}]"}}}'`
+[relayConfig.chainReader.contracts.median]
+contractABI = '[{"anonymous": false,"inputs": [{"indexed": true,"internalType": "address","name": "requester","type": "address"},{"indexed": false,"internalType": "bytes32","name": "configDigest","type": "bytes32"},{"indexed": false,"internalType": "uint32","name": "epoch","type": "uint32"},{"indexed": false,"internalType": "uint8","name": "round","type": "uint8"}],"name": "RoundRequested","type": "event"},{"inputs": [],"name": "latestTransmissionDetails","outputs": [{"internalType": "bytes32","name": "configDigest","type": "bytes32"},{"internalType": "uint32","name": "epoch","type": "uint32"},{"internalType": "uint8","name": "round","type": "uint8"},{"internalType": "int192","name": "latestAnswer_","type": "int192"},{"internalType": "uint64","name": "latestTimestamp_","type": "uint64"}],"stateMutability": "view","type": "function"}]'
+
+[relayConfig.chainReader.contracts.median.configs]
+LatestRoundRequested = {chainSpecificName = 'RoundRequested', readType = 'event'}
+LatestTransmissionDetails = {chainSpecificName = 'latestTransmissionDetails', outputModifications = [
+    { Fields = { LatestAnswer_ = 'LatestAnswer', LatestTimestamp_ = 'LatestTimestamp' }}
```

Comment:
This config isn't being read properly.  I'm guessing that the whole chain reader isn't somehow.  There's a missing param for Type which should make it return an error, but it's passing the test.

Reply:
Where are you seeing that failure?

Reply:
Oh I think I see it in the original now. I guess my initial encoding was lossy and I never recovered it.

Reply:
I see JSON logic for `uknown modifier type` but I do not see a CBOR equivalent.

Reply:
Ah so the JSON logic was active before the PR, since we were using JSON inside the TOML. We can use an equivalent `UnmarshalText` method for TOML.